front/package-lock.json

```diff
@@ -0,0 +1,9550 @@
+{
```

Comment:
We have a `yarn.lock` file (we use that in docker-compose) so there is no reason we should have a NPM package-lock.json file.

Could you remove this?